Makefile

```diff
@@ -44,14 +44,14 @@ version:
 	@echo Go version: $(shell go version)
 
 install:
-	@echo "Installing dgraph ..."; \
-		GOOS=linux GOARCH=amd64 $(MAKE) -C dgraph install; \
+	@echo "Installing Dgraph..."; \
+		$(MAKE) -C dgraph install; \
 
 install_oss oss_install:
-	GOOS=linux GOARCH=amd64 $(MAKE) BUILD_TAGS=oss install
+	$(MAKE) BUILD_TAGS=oss install
 
 uninstall:
-	@echo "Uninstalling dgraph ..."; \
+	@echo "Uninstalling Dgraph ..."; \
```

Comment:
@skrdgraph Matthew added a number of nice touches to the Makefile here (e.g. adding install target parameter).  I thought it would be good to get all these changes to the Makefile in on one PR (removing hard-coded GOOS and GOARCH variables).  This won't break existing CI steps.

Reply:
my earlier comment is not relevant anymore.